emulator-l.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<manifest>
+
+  <include name="base-l-aosp.xml" />
+
+  <default remote="caf" revision="refs/tags/android-5.0.0_r6" sync-j="4"/>
+
+  <!-- Emulator specific things -->
+  <project name="device/generic/armv7-a-neon" path="device/generic/armv7-a-neon"/>
+  <project name="device_generic_goldfish" path="device/generic/goldfish" remote="b2g" revision="b2g-5.0.0_r6"/>
+  <!-- external/qemu for emulator-l need to be updated in bug-1121378 -->
+  <project name="platform_external_qemu" path="external/qemu" remote="b2g" revision="b2g-lollipop"/>
+  <project name="platform/external/wpa_supplicant_8" path="external/wpa_supplicant_8"/>
+  <project name="android-sdk" path="sdk" remote="b2g" revision="b2g-5.0.0_r6"/>
+  <!-- hardware-ril for emulator-l need to be updated in bug-1128837 -->
+  <project name="platform_hardware_ril" path="hardware/ril" remote="b2g" revision="master"/>
```

Comment:
Please use 

 <project name="platform/hardware/ril" path="hardware/ril"/>

for the CAF branch. Our ril daemon doesn't yet build on emulator-l.